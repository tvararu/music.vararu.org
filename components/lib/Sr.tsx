const Sr = ({ children }) => (
  <>
    <style jsx>{`
      span {
        border: 0;
        clip: rect(0 0 0 0);
        height: 1px;
        margin: -1px;
        overflow: hidden;
        padding: 0;
        position: absolute;
        width: 1px;
      }
    `}</style>
    <span>{children}</span>
  </>
);

export default Sr;
