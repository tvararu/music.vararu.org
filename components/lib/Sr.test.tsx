import { cleanup, render } from "@testing-library/react";
import Sr from "./Sr";

afterEach(cleanup);

it("matches snapshot", () => {
  const { asFragment } = render(<Sr>Screen reader content</Sr>);

  expect(asFragment()).toMatchSnapshot();
});
