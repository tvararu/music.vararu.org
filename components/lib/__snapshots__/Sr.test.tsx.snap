// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <span
    class="jsx-140587568"
  >
    Screen reader content
  </span>
</DocumentFragment>
`;
